### ISSUE TITLE
Questions About FT-A and FT-L

### ISSUE THREAD

**Issue by Tot-ziens** [TrustedLLM/UnKE#6]
Hello , I am wondering why the performance of  "FT-A" and "FT-L" is so low . 
In my experience,  methods based on fine-tuning such as SFT and LoRA achieve high performance.
Could you provide the model about "FT-A" and "FT-L" and their training codes.
 Hope for your response.

**Comment by DJC-GO-SOLO** [comment]
As described in the paper, our baseline is implemented based on the EasyEdit library. There are two implementations of their FT:
FT, which only calculates the CE Loss of the last token, which follows the setting of ROME.
FT-M, the normal SFT.
Our FT-L and FT-A are both variants of FT.

About the model: We perform batch_size=1 editing, which is equivalent to saving an editing result after editing once, so the model will not be saved.
About the code, of course, the code needs to be used in conjunction with the easyedit library. Please give me your email address and I will send you the baseline code and usage method. Please note that you should clone the contents of the easyedit library locally before this.

The implementation of Lora is no different from the standard lora. You can refer to the lora results we reported.
